containers/Containerfile

```diff
@@ -4,6 +4,8 @@ RUN git clone https://github.com/wg/wrk.git --depth=1
 RUN cd wrk && make -j $(nproc)
 
 FROM registry.access.redhat.com/ubi8/ubi:latest
+ARG VEGETA_VERSION
 RUN dnf install -y iproute procps-ng
+RUN curl -sS -L https://github.com/tsenart/vegeta/releases/download/v${VEGETA_VERSION}/vegeta_${VEGETA_VERSION}_linux_amd64.tar.gz | tar xz -C /usr/bin/
```

Comment:
Multi-arch container might break here, as we are always pulling `linux-amd64`

Reply:
By the looks of is vegeta is available for amd64/arm64 and not for other archs.

Reply:
Yeah, added a note about the supported archs

Reply:
> Yeah, added a note about the supported archs. Only amd64 for now, we can work to integrate others in the future

Ack, I missed the README.md.

I do not want us to publish/ship multi-architecture images with a linux_amd64 binary,
can we please update this?
https://github.com/cloud-bulldozer/ingress-perf/blob/main/Makefile#L38